tests/ecm_prep_test.py

```diff
@@ -18605,6 +18606,1033 @@ def test_mseg_ok_frefr_co2_map(self):
+                  "Ventilation": {},
+                  "Water Heating": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  }
+                },
+            "efficient": {
+                  "Computers and Electronics": {},
+                  "Cooking": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Cooling (Env.)": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Cooling (Equip.)": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Heating (Env.)": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Heating (Equip.)": {
+                    "Electric": {
+                      "2009": 1.73283767,
+                      "2010": 3.374473358
+                    },
+                    "Non-Electric": {
+                      "2009": 89.17333333,
+                      "2010": 78.65333333
+                    }
+                  },
+                  "Lighting": {},
+                  "Other": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Refrigeration": {},
+                  "Ventilation": {},
+                  "Water Heating": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  }
+                },
+            "savings": {
+                  "Computers and Electronics": {},
+                  "Cooking": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Cooling (Env.)": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Cooling (Equip.)": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Heating (Env.)": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Heating (Equip.)": {
+                    "Electric": {
+                      "2009": -1.7328376703841428,
+                      "2010": -3.374473358116475
+                    },
+                    "Non-Electric": {
+                      "2009": 5.826666666666668,
+                      "2010": 11.346666666666664
+                    }
+                  },
+                  "Lighting": {},
+                  "Other": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  },
+                  "Refrigeration": {},
+                  "Ventilation": {},
+                  "Water Heating": {
+                    "Electric": {},
+                    "Non-Electric": {}
+                  }
+                }
+        }
```

Comment:
Similarly, I wonder if we can move these dicts to another file, just to compress/decrease the length of ecm_prep_test.py. That will hep with readability. No necessary changes needed right now. Maybe a potential issue/PR for the future.